lib/Client.js

```diff
@@ -1387,16 +1387,33 @@ class Client extends EventEmitter {
 
     /**
     * Create a guild
-    * @arg {String} name The name of the guild
-    * @arg {String} region The region of the guild
-    * @arg {String} [icon] The guild icon as a base64 data URI. Note: base64 strings alone are not base64 data URI strings
+    * @arg {Object} options The properties of the guild
+    * @arg {String} options.name The name of the guild
+    * @arg {String} [options.region] The region of the guild
+    * @arg {String} [options.icon] The guild icon as a base64 data URI. Note: base64 strings alone are not base64 data URI strings
+    * @arg {Number} [options.verificationLevel] The guild verification level
+    * @arg {Number} [options.defaultNotifications] The default notification settings for the guild. 0 is "All Messages", 1 is "Only @mentions".
+    * @arg {Number} [options.explicitContentFilter] The level of the explicit content filter for messages/images in the guild. 0 disables message scanning, 1 enables scanning the messages of members without roles, 2 enables scanning for all messages.
+    * @arg {String} [options.systemChannelID] The ID of the system channel
+    * @arg {String} [options.afkChannelID] The ID of the AFK voice channel
+    * @arg {Number} [options.afkTimeout] The AFK timeout in seconds
+    * @arg {Array<Object>} [options.roles] The new roles of the guild, the first one is the @everyone role. IDs are placeholders which allow channel overwrites.
```

Comment:
Maybe we could document the structure of Object? At least a simple explanation in the parameter description

Reply:
Documenting the structure wold be equivalent to documenting an entire role/channel object, which seems excessive. I did add the full structure in typings at least.

Reply:
That's fair, I just thought about mentioning that the structure of the Object is type of Channel or Role.